fdbserver/workloads/ClogRemoteTlog.actor.cpp

```diff
@@ -0,0 +1,161 @@
+#include <cstdint>
+#include "fdbclient/DatabaseContext.h"
+#include "fdbclient/FDBTypes.h"
+#include "fdbclient/Status.h"
+#include "fdbclient/StatusClient.h"
+#include "fdbrpc/PerfMetric.h"
+#include "fdbrpc/SimulatorProcessInfo.h"
+#include "fdbrpc/simulator.h"
+#include "fdbserver/ServerDBInfo.actor.h"
+#include "fdbserver/workloads/workloads.actor.h"
+#include "flow/IPAddress.h"
+#include "flow/IRandom.h"
+#include "flow/Optional.h"
+#include "flow/Trace.h"
+#include "flow/flow.h"
+#include "flow/genericactors.actor.h"
+
+#include "flow/actorcompiler.h" // This must be the last #include.
+
+struct ClogRemoteTLog : TestWorkload {
+	static constexpr auto NAME = "ClogRemoteTLog";
+
+	bool enabled{ false };
+	double testDuration{ 0.0 };
+	double lagMeasurementFrequencySec{ 0 };
+	double clogInitDelaySec{ 0 };
+	double clogDurationSec{ 0 };
+	double lagThresholdSec{ 0 };
+
+	ClogRemoteTLog(const WorkloadContext& wctx) : TestWorkload(wctx) {
+		enabled =
+		    (clientId == 0); // only run this workload for a single client, and that too the first client (by its id)
+		testDuration = getOption(options, "testDuration"_sr, 1000);
+		lagMeasurementFrequencySec = getOption(options, "lagMeasurementFrequencySec"_sr, 5);
+		clogInitDelaySec = getOption(options, "clogInitDelaySec"_sr, 5);
+		clogDurationSec = getOption(options, "clogDurationSec"_sr, 5);
+		lagThresholdSec = getOption(options, "lagThresholdSec"_sr, 5);
```

Comment:
maybe try a larger value, say 60, to see if the commented out assertion still triggers?

Reply:
The current values in TOML file trigger the assert (if uncommented). I should update the default values here to reflect TOML file values, so that even if TOML file does not specify values, the defaults are good enough to reproduce the issue. Makes sense? Or did you have any other concern?